src/mame/matic/barata.cpp

```diff
@@ -339,4 +339,4 @@ ROM_END
 /*************************
 *      Game Drivers      *
 *************************/
-GAME( 2002, barata, 0, barata, barata, barata_state, empty_init, ROT0, "Eletro Matic Equipamentos Eletromec??nicos", "Dona Barata (early prototype)", MACHINE_IMPERFECT_GRAPHICS )
+GAME( 2002, barata, 0, barata, barata, barata_state, empty_init, ROT0, "Eletro Matic Equipamentos Eletromecânicos", "Dona Barata (early prototype)", MACHINE_IMPERFECT_GRAPHICS )
```

Comment:
This won’t work portably because you haven’t added an encoding qualifier for the string.  In particular, MSVC will reject it.

Also, it was incorrectly encoded originally, hence being changed by srcclean.  Have you ensured that it’s correctly encoded as UTF-8 this time?

Reply:
ASCII 0xE2 'â' is encoded as 0xC3 0xA2 in UTF-8.

I've checked out @einstein95's branch and looked up barata.cpp on an hex-editor. It seems to be correct:
![Screenshot from 2022-07-01 17-48-32](https://user-images.githubusercontent.com/213676/176966573-d0932a96-0990-4f90-aaa5-bac9392e9c01.png)

Reply:
But I have also checked out the commit immediately before e6f78d5ed281850d9e8b391c049b8bf696e140bf and in that case the string was already encoded in what seems to be correct UTF-8, as seen below:
![Screenshot from 2022-07-01 18-01-35](https://user-images.githubusercontent.com/213676/176967618-d6dec394-2c96-4130-8aea-cdb46b8cc7a1.png)

So was there something else missing (is there still something else missing?) to ensure that the string is not butchered by the automatic code formatter?

Reply:
It needs to be u8"Eletro Matic Equipamentos Eletromecânicos" because of MSVC I guess.

Reply:
Oh, I was not aware of that! Thanks.

@einstein95, could you please update your commit to use the `u8` prefix on this string, as suggested by @0kmg?

Reply:
> Also, it was incorrectly encoded originally, hence being changed by srcclean. Have you ensured that it’s correctly encoded as UTF-8 this time?

Yes, I've run it through srcclean and it wasn't changed